docs/en/observability/profiling-upgrade.asciidoc

```diff
@@ -19,8 +20,9 @@ To upgrade Universal Profiling, you need to:
 
 . Perform a stack upgrade in {ecloud}
 . Stop profiling data ingestion
-. Delete existing profiling data
-. Setup Universal Profiling from scratch
+. *When upgrading from version 8.8.0 or more recent*: Migrate existing profiling data

```

Comment:
I have a doubt on this one: previously (up until 8.8) we never created the `profiling-metrics` data stream in customers deployment.
When running 8.9 and the new Elasticsearch-based index management, will the data stream be created?

Reply:
Yes, the plugin will create any indices that are missing. Existing component and composable templates will be upgraded to their latest version and the ILM policy has an entirely name (we'll phase out the old one eventually).